src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/PackAllRids.targets

```diff
@@ -29,4 +29,10 @@
 
     <MSBuild Projects="@(ProjectToPublish)" Targets="Pack" BuildInParallel="true" />
   </Target>
+
+  <!--
+    Make sure the _GetPackageFiles target exists, since it is always expected for design time builds, but only
+    defined when ImportNuGetBuildTasksPackTargetsFromSdk is true.
```

Comment:
do you know when this is not true?

Wondering if the import of this file can be conditioned on the existence of it or something - https://github.com/dotnet/roslyn/blob/main/src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj#L142

Reply:
I didn't make a note of this at the time, but it may be related to this:

https://github.com/dotnet/roslyn/blob/e872eb4e0a3b819daa78326e66a792bb7afb8daf/src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/PackAllRids.targets#L4